planning/behavior_path_planner/src/utilities.cpp

```diff
@@ -792,8 +792,8 @@ PathWithLaneId removeOverlappingPoints(const PathWithLaneId & input_path)
       min_dist) {
       filtered_path.points.back().lane_ids.push_back(pt.lane_ids.front());
       filtered_path.points.back().point.longitudinal_velocity_mps = std::min(
-        pt.point.longitudinal_velocity_mps,
-        filtered_path.points.back().point.longitudinal_velocity_mps);
+        std::abs(pt.point.longitudinal_velocity_mps),
+        std::abs(filtered_path.points.back().point.longitudinal_velocity_mps));
```

Comment:
hmm. This seems to cause some problems

Reply:
this is not needed so remove it. https://github.com/autowarefoundation/autoware.universe/pull/1260/commits/2625cad25ab5b05fd412af803a9092aead7bc86a